<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-C4MUU9F" failures="0" tests="11" name="verify_ADMAPPSMSAAgentOrderCancellation_FETCHRLocation_Attempt4" time="1.401" errors="0" timestamp="1 Jun 2019 08:09:16 GMT">
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="@BeforeTest setUp" time="0.0">
    <failure type="org.openqa.selenium.SessionNotCreatedException" message="Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 9.0.0. The following are available: 324774680404 (9)
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:25:53&amp;apos;
System info: host: &amp;apos;DESKTOP-C4MUU9F&amp;apos;, ip: &amp;apos;192.168.43.59&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;11.0.2&amp;apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 9.0.0. The following are available: 324774680404 (9)
    at getResponseForW3CError (C:\Users\Arindam Sarkar\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Users\Arindam Sarkar\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:25:53&amp;apos;
System info: host: &amp;apos;DESKTOP-C4MUU9F&amp;apos;, ip: &amp;apos;192.168.43.59&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;11.0.2&amp;apos;
Driver info: driver.version: AndroidDriver">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 9.0.0. The following are available: 324774680404 (9)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-C4MUU9F', ip: '192.168.43.59', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.2'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 9.0.0. The following are available: 324774680404 (9)
    at getResponseForW3CError (C:\Users\Arindam Sarkar\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Users\Arindam Sarkar\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-C4MUU9F', ip: '192.168.43.59', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.2'
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:38)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:84)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:94)
	at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:95)
	at com.jawwy.modules.Jawwy_ADM_App.setUp(Jawwy_ADM_App.java:67)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
	... 35 more
Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 9.0.0. The following are available: 324774680404 (9)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-C4MUU9F', ip: '192.168.43.59', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.2'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to find an active device or emulator with OS 9.0.0. The following are available: 324774680404 (9)
    at getResponseForW3CError (C:\Users\Arindam Sarkar\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Users\Arindam Sarkar\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="@AfterTest tearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPAgentLogin" time="0.003">
    <skipped/>
  </testcase> <!-- verify_ADMAPPAgentLogin -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPAgentLogin_RememberMeCheck" time="0.002">
    <skipped/>
  </testcase> <!-- verify_ADMAPPAgentLogin_RememberMeCheck -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPAgentOrderSearch_Location1" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPAgentOrderSearch_Location1 -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPAgentOrderSearch_Location2" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPAgentOrderSearch_Location2 -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPDirectSalesAgentOrderSearch" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPDirectSalesAgentOrderSearch -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPSMSAAgentOrderCancellation_Attempt1" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPSMSAAgentOrderCancellation_Attempt1 -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPSMSAAgentOrderCancellation_Attempt2" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPSMSAAgentOrderCancellation_Attempt2 -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPSMSAAgentOrderCancellation_Attempt3" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPSMSAAgentOrderCancellation_Attempt3 -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPSMSAAgentOrderCancellation_Attempt4" time="0.0">
    <skipped/>
  </testcase> <!-- verify_ADMAPPSMSAAgentOrderCancellation_Attempt4 -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPSMSAAgentOrderCancellation_Attempt5" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPSMSAAgentOrderCancellation_Attempt5 -->
  <testcase classname="com.jawwy.modules.Jawwy_ADM_App" name="verify_ADMAPPSMSAAgentOrderSearch_SMSALocation" time="0.001">
    <skipped/>
  </testcase> <!-- verify_ADMAPPSMSAAgentOrderSearch_SMSALocation -->
</testsuite> <!-- verify_ADMAPPSMSAAgentOrderCancellation_FETCHRLocation_Attempt4 -->
